From 6a193804a87a69856c3b5705b24a19a6ba4b6e16 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Tue, 24 Jan 2012 22:51:27 -0500
Subject: [PATCH] add MIN/MAX macros to sys/param.h

this is a nonstandard junk header anyway, so just do what apps expect..
---
 include/sys/param.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/sys/param.h b/include/sys/param.h
index 1ed5337..30499cc 100644
--- a/include/sys/param.h
+++ b/include/sys/param.h
@@ -10,6 +10,11 @@
 #undef MAXPATHLEN
 #define MAXPATHLEN 4096
 
+#undef MIN
+#undef MAX
+#define MIN(a,b) (((a)<(b))?(a):(b))
+#define MAX(a,b) (((a)>(b))?(a):(b))
+
 #include <sys/resource.h>
 #include <endian.h>
 #include <limits.h>
-- 
1.7.3.2

